bundle/config/mutator/process_target_mode.go

```diff
@@ -105,6 +105,12 @@ func transformDevelopmentMode(ctx context.Context, b *bundle.Bundle) diag.Diagno
 		// (registered models in Unity Catalog don't yet support tags)
 	}
 
+	for i := range r.Schemas {
+		prefix = "dev_" + b.Config.Workspace.CurrentUser.ShortName + "_"
+		r.Schemas[i].Name = prefix + r.Schemas[i].Name
+		// (schemas don't yet support tags)
```

Comment:
Note that schemas do support tags.

Reply:
It's not supported via the HTTP API. Only the SQL one. Modified the comment to say as much.